appserver/ejb/ejb-container/src/main/java/org/glassfish/ejb/admin/cli/ListTimers.java

```diff
@@ -146,6 +146,9 @@ public void execute(AdminCommandContext context) {
 
     private String[] listTimers(String[] serverIds) {
         String[] result = new String[serverIds.length];
```

Comment:
Why allocate `new String[...]` just to be discarded `if(nonNull(ejbTimerService))`?